doc/src/sphinx/userguide/installing_clowder.rst

```diff
@@ -0,0 +1,577 @@
+.. _installing_clowder:
+
+##################
+Installing Clowder
+##################
+
+Clowder can be deployed in two ways. Users can either use `Docker <http://docker.com>`_  or manually install the core and
+required dependencies. The :ref:`docker_install` method is the quickest method to run Clowder. It can be used for testing
+features and development. It can be used in production but most of the current production but it hasn't been thoroughly
+tested. Production instances usually use the second method of installation :ref:`manualinstall`. The only exception is that
+most extractors are deployed using `Docker swarm <https://docs.docker.com/engine/swarm/>`_.
+
+What type of user are you?
+===================================
+
+
+For **users of Clowder**: Clowder is easily accessed via Docker. :ref:`Quickstart here<usersOfClowder>`.
+
+For **developers of Clowder**, a hybrid is recommended. :ref:`Dev quickstart here<clowderDevelopers>`.
+
+-  Run the Clowder instance manually via IntelliJ’s Play-2 run
+   configuration.
+-  Run the required services via Docker, and expose each of their ports
+   to Clowder.
+
+For **production instances of Clowder**, a Kubernetes deployment is recommended and manual installations are being phased out.
+
+-  See `Kubernetes install instructions here <https://opensource.ncsa.illinois.edu/confluence/display/CATS/Kubernetes>`__
```

Comment:
Let's point them to https://github.com/clowder-framework/clowder-helm instead. It will contain the most recent information.